[2023-10-06T16:21:59.176+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-10-05T00:00:00+00:00 [queued]>
[2023-10-06T16:21:59.193+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-10-05T00:00:00+00:00 [queued]>
[2023-10-06T16:21:59.195+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-10-06T16:21:59.196+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-10-06T16:21:59.197+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-10-06T16:21:59.226+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-10-05 00:00:00+00:00
[2023-10-06T16:21:59.237+0000] {standard_task_runner.py:55} INFO - Started process 64 to run task
[2023-10-06T16:21:59.246+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-10-05T00:00:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpuhmmfl69']
[2023-10-06T16:21:59.250+0000] {standard_task_runner.py:83} INFO - Job 57: Subtask create_table
[2023-10-06T16:21:59.381+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-10-05T00:00:00+00:00 [running]> on host a8d87f11b01b
[2023-10-06T16:21:59.516+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-10-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-05T00:00:00+00:00
[2023-10-06T16:21:59.535+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-10-06T16:21:59.543+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
            , parameters: None
[2023-10-06T16:21:59.545+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-10-06T16:21:59.561+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20231005T000000, start_date=20231006T162159, end_date=20231006T162159
[2023-10-06T16:21:59.620+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-10-06T16:21:59.670+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
